server/src/interfaces/person.interface.ts

```diff
@@ -41,14 +41,16 @@ export interface IPersonRepository {
   getAllWithoutFaces(): Promise<PersonEntity[]>;
   getById(personId: string): Promise<PersonEntity | null>;
   getByName(userId: string, personName: string, options: PersonNameSearchOptions): Promise<PersonEntity[]>;
+  getManyByName(userId: string, personNames: string[], options: PersonNameSearchOptions): Promise<PersonEntity[]>;
 
   getAssets(personId: string): Promise<AssetEntity[]>;
 
-  create(entity: Partial<PersonEntity>): Promise<PersonEntity>;
+  create(entities: any): any;
```

Comment:
I asked you here https://github.com/immich-app/immich/pull/6455#discussion_r1701686555 :_ )

I can move them to array, but as I commented that will imply more changes in other files and tests, unless there's another way to achieve it I guess

Reply:
You can use a rest parameter if it's easier.

Reply:
> I asked you here [#6455 (comment)](https://github.com/immich-app/immich/pull/6455#discussion_r1701686555) :_ )
> 
> I can move them to array, but as I commented that will imply more changes in other files and tests, unless there's another way to achieve it I guess

If you realize that changing a type breaks stuff, you shouldn't just change it to `any` to avoid the type issue. Just because the type works doesn't mean it'll work at runtime if a function expects *completely different* arguments...

Reply:
Hi @danieldietzler! Thanks for the feedback, I didn't mean to do anything risky. I'm doing my best to provide a solution that meets your standards and can be supported in the long term. I'll revisit it to enforce proper typing.

Reply:
Fixed by https://github.com/immich-app/immich/pull/6455/commits/2b362bbb9d7ed64cbe479b629a594f26043b1129